documentation/adrs/template.md

```diff
@@ -0,0 +1,43 @@
+# [Decision Title]
+
+Proposed by: @author_github_username
+Date: YYYY-MM-DD
+
+## Context
+---
+
+`// Todo: Describe context here`
+
+> What is the current situation or issue that needs addressing?  
+> Why is this decision necessary?  
+> Any relevant background or constraints.
+
+## Decision
+---
+
+`// Todo: Describe the decision here`
+
+> What is the decision being made?  
+> State it clearly and concisely.
+
+## Consequences
```

Comment:
IMO more useful can be pros & cons, especially if there is more than one approach proposed